packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -1373,6 +1379,19 @@ const registerNetworkIpc = (mainWindow) => {
   });
 };
 
+
+const executeRequestErrorHandler = async (request, error) => {
+  if (!request || typeof request.onErrorHandler !== 'function') {
```

Comment:
I recommend to call it `executeRequestOnErrorHandler`.
Also, please remove new line above